docs/excel/excel-data-types-overview.md

```diff
@@ -16,7 +16,12 @@ Prior to the data types addition, the Excel JavaScript API supported string, num
 
```

Comment:
Consider adding a screenshot to this article to help show what data types are. It's one of those features that makes people say "Wow, I didn't know Excel could do that."